cinema_service/settings.py

```diff
@@ -125,16 +130,16 @@
 
 USE_I18N = True
 
-USE_TZ = False
+USE_TZ = True
 
 
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/4.0/howto/static-files/
 
 STATIC_URL = "static/"
 
+MEDIA_ROOT = "/files/media"
```

Comment:
The `MEDIA_ROOT` is set to a fixed path. Consider using `os.path.join(BASE_DIR, 'media')` to make it relative to the project directory, which is more flexible and portable.